server/src/routes/api/v1/genresRouter.js

```diff
@@ -1,5 +1,10 @@
 import express from 'express'
 import { Genre } from '../../../models/index.js'
+
+import objection from 'objection'
+const { ValidationError } = objection
+import cleanUserInput from "../../../services/cleanUserInput.js"
+
```

Comment:
Small ordering change to separate the files that you have written from external sources

```js
import express from 'express'
import objection from 'objection'
const { ValidationError } = objection

import { Genre } from '../../../models/index.js'
import cleanUserInput from "../../../services/cleanUserInput.js"
```